index.js

```diff
@@ -4,6 +4,22 @@
 module.exports = {
   name: 'ember-truncate',
   included: function(app) {
+    var current = this;
+
+    // If the addon has the _findHost() method (in ember-cli >= 2.7.0), we'll just
+    // use that.
+    if (typeof this._findHost === 'function') {
+      app = this._findHost();
```

Comment:
`this._findHost` is private. Let's use the public method [`this.import`](https://github.com/ember-cli/ember-cli/blob/5d64cfbf1276cf1e3eb88761df4546c891b5efa6/lib/models/addon.js#L443-L446) instead.